articles-1.html

```diff
@@ -57,7 +57,7 @@ <h1>All Articles</h1>
 				<p class="subheading">Read articles dedicated to oily skin</p>
 
 				<div class="image-first">
-					<a href="clear-oily-skin.html"><img src="images/clear-oily-skin.jpg"></a>
+					<a href="clear-oily-skin.html"><img src="images/oily/clear-oily-skin.jpg"></a>

```

Comment:
I improved file management by sorting images into appropriate files (e.g. "dry" "oily" etc.) so that they wouldn't be cluttered in one images folder.